core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java

```diff
@@ -140,8 +140,8 @@ public Buildable withOverrides(Map<String,String> overrides) {
       return this;
     }
 
-    @SuppressFBWarnings(value = "URLCONNECTION_SSRF_FD",
-        justification = "location of props is specified by an admin")
+    @SuppressFBWarnings(value = {"URLCONNECTION_SSRF_FD", "PATH_TRAVERSAL_IN"},
+        justification = "location of props is specified by an admin, path provided by test")
```

Comment:
Is this suppression necessary on this method? This method doesn't look like it changed. It might be covered by the suppression elsewhere.